chainstate/storage/src/internal/expensive.rs

```diff
@@ -16,51 +16,56 @@
 use super::*;
 
 use crate::schema::{self as db};
+use common::chain::UtxoOutPoint;
 use storage::MakeMapRef;
+use utxo::Utxo;
 
-impl<B: storage::Backend> Store<B> {
+impl<'a, B: storage::Backend> StoreTxRo<'a, B> {
```

Comment:
Minor: You might be able to elide the lifetime here:

```suggestion
impl<B: storage::Backend> StoreTxRo<'_, B> {
```

Reply:
Done